src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -71,7 +74,12 @@ const getPaddedStyle = (): ICSSInJSStyle => ({
   padding: pxToRem(4),
 })
 
-const getSize = size => pxToRem(sizes.get(size))
+const getSize = (size, sizeModifier) => {
+  if (sizeModifier && size === 'large') {
```

Comment:
would rather suggest to stick to the semantics of this method: 
- if there is no modifier provided, then just `size` should be returned
- otherwise `sizeModifier` should be processed
   - and currently it has specific logic for `large` size values only

```jsx
const getSize = (size, sizeModifier) => {
  if (!sizeModifier) {
     return size
  }

  const modifiedSizes = {
     'large': {
        'x': ...,
        'xx': ...
     } 
  } 

  return modifiedSizes[size] && modifiedSizes[size][sizeModifier]
}
```

This delivers much cleaner intent and will alleviate future refactorings.